src/utils.rs

```diff
@@ -32,12 +32,10 @@ pub(crate) fn from_bstr(string: BSTR) -> String {
 }
 
 pub(crate) fn from_lpstr(string: LPCSTR) -> String {
-    unsafe {
-        let len = (0..).take_while(|&i| *string.offset(i) != 0).count();
+    let len = unsafe { (0..).take_while(|&i| *string.offset(i) != 0).count() };
```

Comment:
You can wrap this even tighter, only `*string.offset(i)` needs to reside in an unsafe block :slightly_smiling_face:.